x-pack/plugins/enterprise_search/server/lib/engines/field_capabilities.ts

```diff
@@ -21,12 +21,24 @@ export const fetchEngineFieldCapabilities = async (
   engine: EnterpriseSearchEngine
 ): Promise<EnterpriseSearchEngineFieldCapabilities> => {
   const { name, updated_at_millis, indices } = engine;
+
+  const availableIndicesList = await availableIndices(client, indices);
+
+  if (Array.isArray(availableIndicesList) && !availableIndicesList.length) {
```

Comment:
question: what happens in this component if availableIndicesList is *not* an array?

Reply:
Cool, in that case you can remove the Array.isArray check here.